src/main/java/net/atos/zac/app/informatieobjecten/converter/RESTInformatieobjectConverter.java

```diff
@@ -380,8 +385,19 @@ public List<RESTEnkelvoudigInformatieobject> convertUUIDsToREST(
             final Zaak zaak
     ) {
         return enkelvoudigInformatieobjectUUIDs.stream()
-                .map(enkelvoudigInformatieobjectUUID -> convertToREST(
-                        drcClientService.readEnkelvoudigInformatieobject(enkelvoudigInformatieobjectUUID), zaak))
+                .map(enkelvoudigInformatieobjectUUID -> {
+                    try {
+                        return convertToREST(
+                                drcClientService.readEnkelvoudigInformatieobject(enkelvoudigInformatieobjectUUID), zaak);
+                    } catch (FoutException e) {
+                        if (e.getFout().getStatus() != 404) {
+                            throw e;
+                        }
+                        LOG.warning(() -> "Document niet gevonden: %s".formatted(enkelvoudigInformatieobjectUUID));
```

Comment:
I think we should log it as an error no? I.e. SEVERE level. Because does this not indicate a serious sync issue between OpenZaak en ZAC that needs to be fixed?

Reply:
Good question. I'm not sure if somebody with access to the logs would have to act on this. Say you remove a document from OpenZaak admin (so far the only way I can reproduce the issue), is that bad? If so, who should act on this? If it's an end user, should we show a message on the front end? What do you think?

Reply:
I'll make it SEVERE for now, maybe this warrants some more discussion later.